dev/CommonStyles/AppBarSeparator_themeresources.xaml

```diff
@@ -0,0 +1,63 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Default">
+            <StaticResource x:Key="AppBarSeparatorForeground" ResourceKey="SystemControlForegroundBaseMediumLowBrush" />
```

Comment:
For Dark/Light, let's go with DividerStrokeColorDefaultBrush.

Reply:
That resource doesn't seem to exist in WinUI 2.x nor generic.xaml, what resource are you referring to? Or do you mean to rename the brush from AppBarSeparatorForeground to DividerStrokeColorDefaultBrush?

Reply:
"DividerStrokeColorDefaultBrush" this is one of the new colours in the Latest visual design.

Reply:
Oh lol, haven't merged this in this branch yet. Thank you @mdtauk, now I know what to do.

Reply:
@teaP That brush won't really work as it is transparent black in dark theme I'm afraid:
![Image showing missing appbar separator](https://user-images.githubusercontent.com/16122379/105409702-2347e800-5c31-11eb-9716-a5d95743ced0.png)

Reply:
I synced with the designer -- that's a known bug that will be fixed shortly. It won't look right at the moment but that is the right brush to use.

Reply:
Alright, moved to the new brush.